index.html

```diff
@@ -248,10 +253,10 @@ <h4>Single-Language Localizable Text Field</h4>
 	<section id="language-defaults">
 	<h4 id="resource_wide_default">Resource-wide Defaults</h4>
 			  
-	<p>When a document contains a number of natural language strings (and particularly if those string are all in the same language), using the localized string representation described above can become inefficient. To reduce the complexity of encoding these strings, specifications can establish a document-level default for language and base paragraph direction. These are separate values, as language does not imply direction. There should still be the ability to override either language or direction on any given string value by using the representation found <a href="#single-linguistic-field">above</a>.</p>
+	<p>When a document contains a number of natural language strings (and particularly if those string are all in the same language), using the localized string representation described above can become inefficient. To reduce the complexity of encoding these strings, specifications can establish a document-level default for language and [=block direction=]. These are separate values, as language does not imply direction. There should still be the ability to override either language or direction on any given string value by using the representation found <a href="#single-linguistic-field">above</a>.</p>
```

Comment:
Done

Reply:
Um. I think it should say 'establish a document-level default for language and string direction.

Also, i'd be inclined to substitute 'resource' for 'document' since this may be a database, rather than a file (?)

Reply:
oh, maybe i saw an older bit of copy - i thought it said 'block' rather than 'string'.  The text you quote looks good.